components/utils/icons.tsx

```diff
@@ -1,37 +1,29 @@
 export const SearchIcon = () => (
   <svg
     xmlns="http://www.w3.org/2000/svg"
-    width="24"
```

Comment:
Opinionated nit: Is there a reason for having the adding of icons in a separate commit and then altering them later on? If there is no documentation/other purpose, I suggest moving these changes to the commit adding these icons and just having them there - this is the convention I'm used to.

Reply:
Hmm, I'm wondering if there is something happening with github, because adding of icons should be included in https://github.com/kaupunginnaiset/alman-akka/pull/19/commits/c67159bf6d7d922609a070f156bf83ae40857c80 🤔 

Unless you mean the changes which I made later, when I realised I want to change the icons to different ones, and that was part of https://github.com/kaupunginnaiset/alman-akka/pull/19/commits/83980a41a0feee1828e2cecbc9db1bbb4061a543 and the reason is only that it's part of improving the navbar from the first, initial thing I made. 

To be honest, I'm a bit of afraid opening the git history and doing changes this big to commits that aren't like one or two steps from the HEAD - I haven't done that too often, and when I have tried, I've messed up about everything 😅  But if you think it's something I should do, I can try it 😄

Reply:
No need to do it now, let's just merge this! That's a team convention I guess, my team has a tradition of doing history cleanups. I can show you some day if you want ❤️

Reply:
Yes, I would very much like that! It's always easier to face one's fears with a friend than alone <3